libs/common/src/vault/models/domain/attachment.spec.ts

```diff
@@ -152,8 +154,8 @@ describe("Attachment", () => {
       expect(actual).toBeInstanceOf(Attachment);
     });
 
-    it("returns null if object is null", () => {
-      expect(Attachment.fromJSON(null)).toBeNull();
+    it("returns undefined if object is null", () => {
+      expect(Attachment.fromJSON(null)).toBeUndefined();
```

Comment:
:pick: TypeScript is showing errors for this line because of the `null` usage. The test is valid, adding a `// @ts-expect-error` will silence the error. 
```suggestion
      // @ts-expect-error Testing null input
      expect(Attachment.fromJSON(null)).toBeUndefined();
```

Reply:
This is also common across other test files. If you do update it here, can you update it across all files?